lib/feedbackComponent/views/feedback_form_view.dart

```diff
@@ -98,4 +99,25 @@ class _FeedbackFormViewState extends ConsumerState<FeedbackFormView> {
       ),
     );
   }
+
+  Widget optionalEmailTextField() {
+    return StreamBuilder(
+      stream: ref.watch(feedbackViewModel).showEmailTextField,
+      builder: (context, snapshot) {
+        if (snapshot.hasData) {
+          return FeedbackTextField(
```

Comment:
Do you think that we should add a helptext that we are going to contact them under this adress and call them by the name specified or would this be apparent?

Reply:
imo this should be apparent 😄 what do you think?

Reply:
Idk, that was why I was asking you ^^